lib/index.js

```diff
@@ -1,4 +1,5 @@
 import 'typeface-roboto';
```

Comment:
Since we are using Inter now, shouldn't we remove Roboto?

Reply:
Ok, good idea. It's now done.